.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -118,110 +113,78 @@ jobs:
+        echo "Running multi-bundle updater for version ${{ steps.split.outputs.version }} with manual URL ${{ needs.prepare-environment.outputs.manual_url }}..."
+        scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" --split bundles
         zip vulns/vulnerabilities.zip bundles/*.json.zst
 
-    - uses: ./.github/actions/upload-artifact-with-retry
-      with:
-        name: vulns
-        path: vulns
-        if-no-files-found: error
-
-  upload-vulnerabilities:
-    runs-on: ubuntu-latest
-    needs:
-    - run-updater
-    steps:
-    # Checkout to run ./.github/actions/download-artifact-with-retry
-    - name: Checkout repository
-      uses: actions/checkout@v4
-      with:
-        fetch-depth: 0
-        ref: ${{ github.event.pull_request.head.sha }}
-
-    - uses: ./.github/actions/download-artifact-with-retry
-      with:
-        name: vulns
-        path: vulns
-
     - name: Authenticate with Google Cloud
-      if: github.event_name == 'schedule'
       uses: google-github-actions/auth@v2
       with:
         credentials_json: ${{ secrets.GOOGLE_SA_STACKROX_HUB_VULN_DUMP_UPLOADER }}
 
-    - name: Authenticate with Google Cloud
-      if: github.event_name == 'pull_request'
-      uses: google-github-actions/auth@v2
-      with:
-        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
-
     - name: Set up Cloud SDK
       uses: google-github-actions/setup-gcloud@v2
 
-    - name: Update dev vulnerabilities
-      if: github.event_name == 'schedule'
+    - name: Upload vulnerabilities to specific path
       run: |
-        cd vulns
-        # Upload development tags for upstream (dev) and downstream (1.0.0).
-        gsutil -m cp -r . gs://definitions.stackrox.io/v4/vulnerability-bundles/dev/
-        gsutil -m cp -r . gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/
+        gsutil -m cp vulns/* gs://definitions.stackrox.io/v4/vulnerability-bundles/${{ steps.split.outputs.version }}/
 
-    - name: Update PR vulnerabilities
-      if: github.event_name == 'pull_request'
+    - name: Upload vulnerabilities to downstream path
+      if: github.event_name == 'schedule' && steps.split.outputs.version == 'dev'
       run: |
-        branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}
-        # Replace / with -, so the branch name isn't truncated when pushed to GCS.
-        dir=${branch////-}
-        case $dir in
-        dev|1.0.0)
-          echo "Branch $dir is disallowed"
-          exit 1
-        esac
-        cd vulns
-        gsutil cp -r . "gs://scanner-v4-test/vulnerability-bundles/$dir"
+        echo "Uploading vulnerabilities to downstream path: gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/"
+        gsutil -m cp vulns/* gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/
 
   send-notification:
     needs:
-    - upload-vulnerabilities
+    - build-and-upload
     runs-on: ubuntu-latest
     if: failure()
```

Comment:
nit: perhaps expand on the `failure()` condition here to not send a notification when this is triggered outside of a schedule (so that we don't get notifications from PRs/dispatches, or perhaps add to the message for the notification to make it clear that whomever is support doesn't have to waste cycles looking into the failure)